src/app/public/modules/text-expand/text-expand.component.ts

```diff
@@ -292,6 +292,7 @@ export class SkyTextExpandComponent implements AfterContentInit {
     this.buttonText = expanding ? this.seeLessText : this.seeMoreText;
     // Measure the new height so we can animate to it.
     let newHeight = adapter.getContainerHeight(container);
+    /* istanbul ignore if */
```

Comment:
☝️ Has nothing to do with this PR. The change to the code base was just enough to slip this under coverage by .01%, causing the build to fail. Had to go digging through old components to find a way to make up for it. This line is actually hit by a test so I'm not sure why its not showing as covered. Regardless, I figured the urgency of this fix outweighed taking the time to write a niche unit test that covers a visual nicety.